docs/pages/application-access/okta/scim-only.mdx

```diff
@@ -0,0 +1,166 @@
+---
+title: Installing SCIM-only Okta integration
+description: How to install a SCIM-only Okta integration
+---
+
+## Before you begin
+This test involves upgrading to a dev build of Teleport 15 - back up your
+cluster config & back-end data just in case.
```

Comment:
I would remove this, the goal for this is to just become a regular guide in the docs. I will share the dev build with interested customers as needed before this makes it into an actual release.